geonode/security/utils.py

```diff
@@ -523,6 +523,9 @@ def safe_remove(perms, perm): perms.remove(perm) if perm in perms else None
                     if not AdvancedSecurityWorkflowManager.is_simple_publishing_workflow() and (_resource.is_approved or _resource.is_published):
                         safe_remove(prev_perms, 'change_resourcebase')
                         safe_remove(prev_perms, 'change_resourcebase_metadata')
+                        if _resource.polymorphic_ctype.model == "dataset":
+                            safe_remove(prev_perms, 'change_layer_style')
```

Comment:
Those perms in `master` have a different name:

- `change_dataset_style`
- `change_dataset_data`

Reply:
Fixed, thanks!